src/Comments/Comment.jsx

```diff
@@ -275,7 +267,7 @@ return (
                   src="${REPL_ACCOUNT}/widget/Comments.Compose"
                   props={{
                     initialText: `@${accountId}, `,
-                    notifyAccountId: extractNotifyAccountId(state.content.item),
+                    notifyAccountId: accountId,
```

Comment:
I'm worried this change might break the notification that's sent to the post owner when someone creates a new comment. We need to make sure that:

1. New comments notify the original post author that someone left a comment on their post.
2. A comment that is liked sends a notification to the comment author.

Reply:
I've reverted the changes regarding the `extractNotifyAccountId` function.